### ISSUE TITLE
error in setting mlpack local environment for jupyter 

### ISSUE THREAD

**Issue by kumar-utkarsh0317** [mlpack/examples#218]
I am providing a picture of the error that I am getting, and I have just no clue why this error is occurring.

![Screenshot from 2024-01-05 01-31-04](https://github.com/mlpack/examples/assets/147777151/54744c6d-8b1a-4fcc-8aa9-1ccf420ddfbe)

**Comment by zoq** [comment]
Can you run:

```
conda activate notebook
cd examples
jupyter lab
```

In this example I expect that the conda env is called `notebook`.

**Comment by kumar-utkarsh0317** [comment]
> Can you run:
> 
> ```
> conda activate notebook
> cd examples
> jupyter lab
> ```
> 
> In this example I expect that the conda env is called `notebook`.

No, I am not able to run this command because the 'jupyter-lab' package has not been installed by the script in this notebook environment. Let me elaborate in detail on how I am proceeding.

Step 1: First, to run the script, I am using the command:
```
chmod +x jupyter-conda-setup.sh
./jupyter-conda-setup.sh notebook
```
The script downloads many packages, but at the end, I receive a setup unsuccessful message."

**Comment by kumar-utkarsh0317** [comment]
Okay, I have proceeded a little bit, but the program has stopped at this point. I am providing a picture of the terminal. Please tell me, do I have to install all those packages separately, or is it taking time to install all those packages?

![Screenshot from 2024-01-05 08-56-53](https://github.com/mlpack/examples/assets/147777151/b2132340-e86e-436d-9d84-8f25e31e95a2)

**Comment by kumar-utkarsh0317** [comment]
I have installed all the above packages separately, but now I am getting an error indicating that the Python version is incompatible with the ipykernel.

![Screenshot from 2024-01-07 11-23-58](https://github.com/mlpack/examples/assets/147777151/48936346-e8a8-46d7-ab1b-c9fcf11f93ab)

**Comment by zoq** [comment]
I'm updating the env, so it works with a newer version of python.

**Comment by kumar-utkarsh0317** [comment]
Do I need to update my python environment oor you are making some changes
in the script file

**Comment by zoq** [comment]
I'll make changes to the script itself.
